File: frontend/src/dtos/broker/brokerInvitationDto.ts
export interface BrokerInvitationDto {
  id: string
  token: string
  inviteeId: string
  inviterId: string
  inviterEmail: string
  brokerName: string
  expiresAt: string
  createdAt: string
  isAccepted: boolean
  brokerId: string
}

File: backend/ResQueue/ResQueue/Dtos/Broker/BrokerInvitationDto.cs
namespace ResQueue.Dtos.Broker;

public class BrokerInvitationDto
{
    public string Id { get; set; } = null!;
    public string BrokerId { get; set; } = null!;
    public string InviterId { get; set; } = null!;
    public string InviteeId { get; set; } = null!;
    public string InviterEmail { get; set; } = null!;
    public string Token { get; set; } = null!;
    public string BrokerName { get; set; } = null!;
    public DateTime CreatedAt { get; set; }
    public DateTime ExpiresAt { get; set; }
    public bool IsAccepted { get; set; }
}